Stack/Opc.Ua.Core/Types/Encoders/JsonEncoder.cs

```diff
@@ -266,6 +267,15 @@ protected virtual void Dispose(bool disposing)
                 if (m_writer != null)
                 {
                     Close();
+                    // only dispose if the memorystream is owned
+                    // by JsonEncoder instance
+                    if (m_destination != null)
+                    {
+                        m_writer?.Dispose();
+                        m_writer = null;
+                        m_destination.Dispose();
```

Comment:
Is this really necessary? m_writer?.Dispose() should internally dispose the memory stream

Reply:
ok, I wasn't sure...